app/weeklyPicks/WeeklyPicks.tsx

```diff
@@ -171,11 +172,9 @@ const WeeklyPicks = ({ NFLTeams, currentGameWeek }: Props): JSX.Element => {
   }
 
   return (
+    //className="pb-8 text-center text-[2rem] font-bold text-white"
     <section className="w-full pt-8" data-testid="weekly-picks">
-      <h1 className="pb-8 text-center text-[2rem] font-bold text-white">
-        Your pick sheet
-      </h1>
-
+      <Heading heading="h1">Your pick sheet</Heading>
```

Comment:
I'd change this back to what it was prior so that we can make another ticket to replace all existing headers with the new `Heading` component upon approval